### ISSUE TITLE
livenessProbe and readinessProbe not including proxyBasePath

### ISSUE THREAD

**Issue by bug-c** [Unleash/helm-charts#78]
### Describe the bug

When deploying the chart with:
```
env:
  - name: PROXY_BASE_PATH
    value: features
```

The livenessProbe and livenessProbe are set to /proxy/health instead of expected /features/proxy/health. This result in pod been marked as down.

### Steps to reproduce the bug

1. Deploy  chart using PROXY_BASE_PATH=features and an ingress controller with path=features
2. Wait for chart to de deployed
3. Check deployed pod and notice the values of livenessProbe.path and livenessProbe.path
```
        livenessProbe:
            path: /proxy/health
        readinessProbe:
          httpGet:
            path: /proxy/health
```

### Expected behavior

```
        livenessProbe:
            path: /features/proxy/health
        readinessProbe:
          httpGet:
            path: /features/proxy/health
```

### Unleash version

v0.13.1

### Subscription type

Open source

### Hosting type

Self-hosted

**Comment by chriswk** [comment]
Hi @bug-c. You're right that this won't work. There's no connection between a custom defined environment variable and the health checks. 

My suggestion is to extend the proxy configuration to add a `basePath` configuration variable for this purpose. This could then be linked with both the environment variable and the healthchecks.

 I'm assuming it's the `unleash-proxy`.

**Comment by chriswk** [comment]
Fixed with #76 and available through 0.4.0 of the helm chart
